src/providers/node/hash.ts

```diff
@@ -0,0 +1,105 @@
+import * as crypto from 'crypto';
+import Promise from 'dojo-core/Promise';
+import { Binary, Codec, Data, Hasher, HashFunction } from '../../crypto';
+
+/**
+ * Algorithms available through this provider. This
+ */
+export const ALGORITHMS = {
```

Comment:
It was intended to be informational, but for general usage that's probably not all that useful. I'll un-export it for the time being.